interfaces/Portal/src/app/components/registration-activity-detail/registration-activity-detail.component.html

```diff
@@ -14,7 +14,7 @@
       class="activity-detail--header util--flex-row ion-justify-content-between ion-align-items-baseline"
     >
       <span>
-        <strong>
+        <strong data-testid="activity-detail-message-title">
```

Comment:
I think this slipped through the cracks and wasn't meant to be here (because it isn't being used anywhere yet). Ideally you would remove this from the PR but not a big deal if you know you are going to end up using it eventually.

Reply:
Yes I just kept it here to have an example but I'll address it with another PR where I want to change CSS locators with test IDs